tests/unit/test_mcp_gateway.py

```
@@ -281,3 +281,807 @@ def test_empty_allowed_tools_fails(self, tmp_path):
+        assert "debug_event_2" in captured.err
+
+    def test_does_not_emit_secrets(self, capsys):
+        from mcp_gateway.audit.logger import AuditLogger
+
+        log = AuditLogger()
+        # シークレットがマスクされることを検証
+        log.log(
+            ev="call",
+            agent="a",
+            api_key="sk-hidden",
+            ck_token="secret",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "ck_token"

(S106)